pages/instructorQuestions/instructorQuestions.ejs

```diff
@@ -1,136 +1,253 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
+  
+  <head>
     <%- include('../partials/head'); %>
-    <link href="<%= node_modules_asset_path('tablesorter/dist/css/theme.bootstrap.min.css') %>" rel="stylesheet" />
-    <script src="<%= node_modules_asset_path('tablesorter/dist/js/jquery.tablesorter.min.js') %>"></script>
-    <script src="<%= node_modules_asset_path('tablesorter/dist/js/jquery.tablesorter.widgets.min.js') %>"></script>
-</head>
+    <script src="<%= node_modules_asset_path('lodash/lodash.min.js') %>"></script>
+    <script src="<%= node_modules_asset_path('bootstrap-table/dist/bootstrap-table.min.js') %>"></script>
+    <script src="<%= node_modules_asset_path('bootstrap-table/dist/extensions/sticky-header/bootstrap-table-sticky-header.min.js') %>"></script>
+    <script src="<%= node_modules_asset_path('bootstrap-table/dist/extensions/filter-control/bootstrap-table-filter-control.min.js') %>"></script>
+    <link href="<%= node_modules_asset_path('bootstrap-table/dist/bootstrap-table.min.css') %>" rel="stylesheet" />
+    <link href="<%= node_modules_asset_path('bootstrap-table/dist/extensions/sticky-header/bootstrap-table-sticky-header.min.css') %>" rel="stylesheet" />
+    <link href="<%= node_modules_asset_path('bootstrap-table/dist/extensions/filter-control/bootstrap-table-filter-control.min.css') %>" rel="stylesheet" />
+  </head>
+  
+  <body>
+    <script>
+      $(function() {
+          $('#questionsTable').bootstrapTable({
+              icons: {
+                  columns: 'fa-th-list',
+              },
+              buttons: {
+                  clearFilters: {
```

Comment:
This button doesn't seem to do anything when I click it. The current filters all stay there with no changes. No errors on the browser JS console.

Reply:
I'd be fine with using the built-in clear-filters button (tooltip is not that important).

Reply:
Fixed.